src/vue/components/morph.vue

```diff
@@ -269,6 +269,8 @@ export default {
   },
   computed: {
     lexemes () {
+      // A call to `defDataReady` will force this computed prop to recalculate every time definitions data is updated
+      let defs = this.$store.getters['app/defDataReady']
```

Comment:
needs to be app/shortDefDataReady now, I believe?

Reply:
Oh, that's correct, fixed that